tasks/level-1/1.7.1.1.yml

```diff
@@ -8,11 +8,29 @@
 # generate motd, and which are related to system information which would otherwise be
 # displayed by mingetty options, are not present.
 # On amazon linux, this is the 30-banner script within /etc/update-motd.d/
-- name: 1.7.1.1 - Ensure mingetty options are not used within motd
+- name: 1.7.1.1 - Ensure message of the day doesn't print sensitive information
   file:
-    path: "/etc/update-motd.d/30-banner"
+    path: "/etc/update-motd.d"
```

Comment:
This probably wouldn't impact us, but I don't think you'd necessarily want to remove the entire directory if sending these changes upstream.

I'd just ensure the following files are removed:

```
/etc/update-motd.d/30-banner
/etc/update-motd.d/50-amazon-linux-extras-news
/etc/update-motd.d/70-available-updates
```